src/commands/workflow/run.ts

```diff
@@ -21,13 +22,18 @@ export default class WorkflowRun extends BaseCommand {
     actor: Flags.string(),
     tenant: Flags.string(),
     data: jsonStr(),
+    "run-local": Flags.boolean({ default: false }),
```

Comment:
Since the command itself is "run", I'm thinking we could make this flag simply `--local`? i.e. `workflow run foo --local`

What do you think?

Reply:
ah yeah that makes more sense!